TheMET/TheMET/ViewControllers/CatalogViewController.swift

```diff
@@ -125,8 +125,7 @@ class CatalogViewController: UIViewController {
                 completion(catalogCellData)
                 return
             }
-            let totalObjectsUInt: UInt = UInt(objects.total)
-            let subtitle: String = (self?.objectsCount(count: totalObjectsUInt))!
+            let subtitle: String = (self?.cellSubtitle(objectsCount: objects.total))!
```

Comment:
force-unwrapping using `!` must not be used.  Instead you can unwrap `self` itself using `guard` on the previous line